net/udprelay/server.go

```diff
@@ -450,6 +452,25 @@ func (c *singlePacketConn) WriteBatchTo(buffs [][]byte, addr netip.AddrPort, gen
 	return nil
 }
 
+// UDP socket read/write buffer size (7MB). At the time of writing this value
```

Comment:
Since the comment mentions "At the time of this writing" can you please also include a date? Blame will tell us the last time it was touched but can drift, and is an extra step to find it.

Reply:
Added.